idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/settings/PersistentSettings.kt

```diff
@@ -23,6 +23,7 @@ class PersistentSettings : SimplePersistentStateComponent<ValkyrieState>(Valkyri
         var nestedPacks by string()
 
         var generatePreview by property(false)
+        var outputFormat by string()
```

Comment:
```suggestion
        var outputFormat: String? by string()
```

Reply:
I see, we should declare this nullable type explicitly.

Reply:
the same)